src/androidTest/java/androidx/os/AsyncTaskTest.kt

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (C) 2018 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.os
+
+import android.os.AsyncTask
+import org.junit.Assert
+import org.junit.Test
+
+/**
+ * Created by muhannad on 2/16/18.
+ */
+class AsyncTaskTest {
+
+    @Test
+    fun testtask() {
+        println("main , Thread : ${Thread.currentThread().name}")
+        val execute = AsyncTaskKTX<Int, Int, Int>().runInBackground {
```

Comment:
I don't like this API: you create an object, then call two methods on it. I'm not sure what this buys you compared to just creating an object that overrides the appropriate methods? At the very least creating the asynctask could be done in runInBackground.

Speaking of which, names: AsynctaskKTX isn't a good name and runInBackground seems to indicate that it will run the task, but it doesn't.